### PR TITLE
Upgrade to go1.12

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

File: manager/resource.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package manager
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (pires) [inline manager/resource.go]: This renders already to `v1` so why the redundant alias? I suggest changing it to `corev1`, which can be found in this repo too, sometimes.

I also suggest we have a separate PR that normalizes this across the entire code base.

Comment (cpuguy83) [reply manager/resource.go]: goimports often does this.

Comment (sargun) [reply manager/resource.go]: This came due to upgrading from go1.10 to go1.12 goimports. Go imports changed their style in 1.11 IIRC where in some cases it makes the alias explicit versus implicit.

Comment (sargun) [reply manager/resource.go]: @pires I wanted to make sure that the version of Go and the Go coding style were upgraded in line.

Comment (pires) [reply manager/resource.go]: OK I didn't know about the behavior you describe. It's OK then.

**Pull request merged**